src/components/shared/MarkerTooltipContents.js

```diff
@@ -157,13 +160,13 @@ function getMarkerDetails(data: MarkerPayload): React.Element<any> | null {
                   'gctime',
                   'Total slice times',
                   timings.total_time,
-                  x => x + 'ms'
+                  x => formatMilliseconds(x, 3, 2)
                 )}
                 {_markerDetail(
                   'gcmaxpause',
                   'Max Pause',
                   timings.max_pause,
-                  x => x + 'ms'
+                  x => formatMilliseconds(x, 3, 2)
```

Comment:
I don't really like that we have these numbers without really understanding what they mean without reading the function definition. But the alternative is using an object parameter for this and I don't want to incur this cost for `formatNumber` that's used quite a lot. So maybe add a comment before the magic numbers, like this:
```javascript
x => formatMilliseconds(x, /* significantDigits */ 3, /* maxFractionalDigits */ 2);
```